docs/source/references/api_spec.md

```diff
@@ -382,17 +381,3 @@ codec's [GitHub repository](https://github.com/paritytech/parity-scale-codec).
 ## Reference Iroha Client Implementation
 
 [Iroha client in Rust.](../../../client)
-
-## Iroha Structures
```

Comment:
You've removed the section, but there are still links to it, i.e. `#iroha-structures`